src/picosvg/svg.py

```diff
@@ -324,6 +344,7 @@ def _inherit_nondefault_overflow(attrib, child, attr_name):
             "clip-rule": _inherit_copy,
             "fill": _inherit_copy,
             "fill-rule": _inherit_copy,
+            "style": _inherit_copy,
```

Comment:
Don't you have to worry about merging styles? Potentially better to apply them and then merge post-apply?

Reply:
i'm not sure what you mean by "merge" here.. Do you mean if the parent group has style attribute while the child has not (and would thus inherit it) but at the same time the child has one of the attributes that the parent group's style defines with inlined css? I can see how that could end up with potentially clashing definitions in the child.
Note however that, at least for `topicosvg()` we are first applying the inlined style attributes and then ungrouping. So the concern above would only apply if one calls ungroup() before apply_style_attributes.